ui/src/shared/components/TimeMachine/TimeMachineControls.tsx

```diff
@@ -39,6 +47,13 @@ const TimeMachineControls: SFC<Props> = ({
         isDynamicSourceSelected={isDynamicSourceSelected}
         onSelectDynamicSource={onSelectDynamicSource}
       />
+
+      <div style={{padding: 15}}>
```

Comment:
Try to avoid inline styles wherever possible unless we need to dynamically control them with javascript.

Reply:
will do!